_includes/layout/base/pagetop-donation.html

```diff
@@ -0,0 +1,71 @@
+{% comment %}
+This file is licensed under the MIT License (MIT) available on
+http://opensource.org/licenses/MIT.
+{% endcomment %}
+
+<div class="donation-container">
+  <div class="donation-text">
+    <p>
+      {% translate donation-banner-text layout %}
+    </p>
+    <button class="donation-btn" onclick="openDonationModal()">
+      {% translate donation-banner-donate-button layout %}
+    </button>
```

Comment:
Yes, it's used in a handful of pages and in this scenario is used as an entry point for loading BTC/USD rates and setting up event handlers.